scripts/yymnist_txtdetection2coco.py

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env python3
```

Comment:
Does this script go away since the make_data now makes the annotations file?

Reply:
Yes, removed this from the branch here af19a16